File: Application/ARQuizServer/entities/Location.php
<?php
/**
 * Created by PhpStorm.
 * User: Martin
 * Date: 13.2.2017
 * Time: 21:24
 */

namespace entities;


class Location
{
    private $id;
    private $block;
    private $floor;

    function __construct($id, $block, $floor)
    {
        $this->id = $id;
        $this->block = $block;
        $this->floor = $floor;
    }

    public function getBlock()
    {
        return $this->block;
    }

    public function getFloor()
    {
        return $this->floor;
    }

    public function getId()
    {
        return $this->id;
    }
}

File: Application/ARQuiz/app/src/main/java/com/lepko/martin/arquiz/Entities/Location.java
package com.lepko.martin.arquiz.Entities;

/**
 * Created by Martin on 25.2.2017.
 */

public class Location {

    private int id;
    private char block;
    private int floor;

    public Location(int id, char block, int floor) {
        this.id = id;
        this.block = block;
        this.floor = floor;
    }

    public int getId() {
        return id;
    }

    public char getBlock() {
        return block;
    }

    public int getFloor() {
        return floor;
    }

    public String getLocation() {return Character.toString(block) + Integer.toString(floor); }
}
